resources/monitoring/charts/prometheus-pushgateway/README.md

```diff
@@ -0,0 +1,92 @@
+# Prometheus Pushgateway
```

Comment:
We don't put readmes for the subcharts as they can be irrelevant for our use case. Please delete this file.